src/coreclr/jit/codegencommon.cpp

```diff
@@ -8933,7 +8933,7 @@ void CodeGen::genFnEpilog(BasicBlock* block)
             fCalleePop = false;
 
 #ifdef UNIX_X86_ABI
-        if (IsCallerPop(compiler->info.compMethodInfo->args.callConv))
+        if (IsCallerPop(compiler->info.compCallConv))
             fCalleePop = false;
```

Comment:
```suggestion
        fCalleePop = false;
```
This method does not handle unmanaged epilogs for Windows x86 today, so we do not need to bother to handle them for Unix x86 either.

Also, `IsCallerPop` can be deleted since this seems to be the only use.

Reply:
I was planning on experimenting with moving more of the Windows x86 handling into the JIT after I got this PR in when I have a chance, so I'd rather leave this for now.